test/client_test.py

```diff
@@ -26,7 +27,7 @@ def test_client_type(servicer, client):
 
 def test_client_platform_string(servicer, client):
     client.hello()
-    platform_str = servicer.last_metadata["x-modal-platform"]
+    platform_str = urllib.parse.unquote(servicer.last_metadata["x-modal-platform"])
```

Comment:
Is it worth mocking some sort of weird unicode output from the `platform` module here?

Reply:
Yes good point, added [an explicit test](https://github.com/modal-labs/modal-client/pull/3158/files#diff-c3f9238c73a319c67ee49a92bcd534772c62b66a7a3c3b61da1ad630d178488bR37) now!